modules/tictactoe/tictactoe.js

```diff
@@ -215,22 +217,21 @@ exports.run = function(api, event) {
         api.sendMessage(output, event.thread_id);
     }
 
-    //If we are trying to make a move
-    if (command.startsWith(commandPrefix + "move")) {
-        var offset = commandPrefix.length + "move".length + 1,
+    // If we are trying to make a move
+    if (command.startsWith(commandPrefix + 'move')) {
+        var offset = commandPrefix.length + 'move'.length + 1,
             move = command.substr(offset),
             game = findGame(event.thread_id, event.sender_name.trim());
 
         if (!game) {
-            api.sendMessage("You're not even playing " + event.sender_name.trim(), event.thread_id);
+            api.sendMessage('You\'re not even playing ' + event.sender_name.trim(), event.thread_id);
             return;
         }
 
         if (move.length !== 2) {
-            api.sendMessage("Um. Is that even a coordinate?");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529976290/issues/source?bid=3282987&fileBranchId=3282998#l232)